src/index.html

```diff
@@ -1,12 +1,345 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="catalog-page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="logo">
+          <img src="images/logo.png" alt="Logo">
+        </a>
```

Comment:
logo should be an element of `header` block.

```suggestion
        <a href="#home" class="header__logo">
          <img src="images/logo.png" alt="Logo">
        </a>
```